File: Backend/src/main/java/com/resumebuilder/backend/dto/DocumentDTO.java
package com.resumebuilder.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DocumentDTO {

    private UUID id;
    private Long userId;
    private String jobTitle;
    private String companyName;
    private String jdText;
    private String resumeMd;
    private String status;
}

File: app/src/main/java/com/snehil/cvoptima/data/remote/model/DocumentDto.kt
package com.snehil.cvoptima.data.remote.model

import com.google.gson.annotations.SerializedName

data class DocumentDto(
    @SerializedName("id") val id: String? = null,
    @SerializedName("userId") val userId: Long? = null,
    @SerializedName("jobTitle") val jobTitle: String? = null,
    @SerializedName("companyName") val companyName: String? = null,
    @SerializedName("jdText") val jdText: String? = null,
    @SerializedName("resumeMd") val resumeMd: String? = null,
    @SerializedName("status") val status: String? = null
)
